src/Lecture1.hs

```diff
@@ -104,7 +113,9 @@ and finds a sum of the numbers inside this string.
 
 The string contains only spaces and/or numbers.
 -}
-strSum str = error "TODO"
+strSum :: String -> Int
+strSum str = sum ints
+    where ints = map (\x -> read x :: Int) (words str)
```

Comment:
As you specify the top-level type signature for this function, the compiler would be able to figure out what type `read` function has in this case due to type inference, so you can skip specifying it in the body:

```suggestion
    where ints = map read (words str)
```